client/src/components/graphs/OrganizationalChart.js

```diff
@@ -78,26 +75,234 @@ const sortPositions = (positions, truncateLimit) => {
     : allResults
 }
 
-// TODO: enable once innerhtml in svg is polyfilled
-// const EXPAND_ICON = renderBlueprintIconAsSvg(IconNames.DIAGRAM_TREE)
-// const COLLAPSE_ICON = renderBlueprintIconAsSvg(IconNames.CROSS)
+const PositionNode = ({ position, size, minified, twoRows }) => {
+  const history = useHistory()
+  const personText = `${position.person ? position.person.rank : ""} ${
+    position.person ? position.person.name : "unfilled"
+  }`
+  const positionText = `${position.name}`
+
+  return (
+    <Tooltip
+      key={position.uuid}
+      popoverClassName="bp3-dark"
+      content={
+        <>
+          <LinkTo modelType="Person" model={position.person} /> -{" "}
+          <LinkTo modelType="Position" model={position} />{" "}
+        </>
+      }
+      targetTagName="g"
+      wrapperTagName="g"
+    >
+      <g onClick={() => history.push(Position.pathFor(position))}>
+        <image
+          width={size[1]}
+          height={size[1]}
+          y={-10}
+          href={position.person?.avatar || DEFAULT_AVATAR}
+        />
+        {!minified && (
+          <>
+            <Text
+              x={size[1] + 2}
+              fontSize="9px"
+              fontWeight={twoRows ? "bold" : "normal"}
+              fontFamily="monospace"
+              textAnchor="start"
+            >
+              {utils.ellipsize(
+                twoRows ? `${personText}` : `${personText} ${positionText}`,
+                (size[0] - size[1]) * 0.17
+              )}
+            </Text>
+            {twoRows && (
+              <Text
+                x={size[1] + 2}
+                y={size[1] / 2}
+                fontSize="9px"
+                fontWeight={twoRows ? "bold" : "normal"}
```

Comment:
Please fix this warning:
![Screenshot from 2020-08-26 14-29-12](https://user-images.githubusercontent.com/445238/91303623-947d8f00-e7a8-11ea-9b4e-9e8f9e24f4f2.png)